### DISCUSSION TITLE
一人血书改回原版UI界面！！

### DISCUSSION THREAD

**Discussion by caonima-bi** [share121/inter-knot#4501] [其他]
![Uploading 356853455-8297507f-6120-476a-b57f-661aca8bff6a.png…]()
为什么改UI了呀，我觉得原本挺还原的，如果是为了和米哈游切割的话完全没必要啊！无论是不是绝区零的玩家都会为了这种二创的生命力感到惊喜，改了之后反而让我们觉得很难受很失落。如果是为了版权的话也没有必要，除非有盈利项目，而且我觉得绳网这个概念米哈游并没有版权的说法吧，如果有那第一个写修仙小说的不得亏死了。总之，真的真的很想念之前的界面，老用户不习惯，新用户也很失望，真的不值得。（认同请顶我上去，让官方看到）

**Comment by The-Place-Where-Winter-Sieeps** [comment]
+1
我就是因为绳网界面进来的，虽然现在的也还行。但我还是希望改回原来的UI。球球了！！！
<img src="https://img-static.mihoyo.com/communityweb/upload/ee7bb800b0c90f66fa9e321cb6d8ee17.png" alt="抱大腿" width="75" ><img src="https://img-static.mihoyo.com/communityweb/upload/ee7bb800b0c90f66fa9e321cb6d8ee17.png" alt="抱大腿" width="75" ><img src="https://img-static.mihoyo.com/communityweb/upload/ee7bb800b0c90f66fa9e321cb6d8ee17.png" alt="抱大腿" width="75" >

**Comment by Jellyfish-DemonKing** [comment]
+1
来玩是因为叫绳网

**Comment by nostalgiatan** [comment]
嗯，不过现在不是改回，是重做。
会一点一点还原的，敬请期待～

**Comment by share121** [comment]
由于新版绳网是从底层上的大改，连编程语言都换了，所以搞好还需要时间。并且我周一到周五还要上学，时间真不多。

**Reply by Dave-12138** [reply]
加个让用户自行注入css的设置功能？这样想要什么样式都可以自己写（<img src="https://bbs-static.miyoushe.com/static/2024/09/04/70420b7df8a3934cb8e7b0f0f4136511_7657550741734618773.png" alt="卡齐娜-囤货" width="75" >

**Reply by share121** [reply]
这个没法实现，因为现在已经完全脱离浏览器，所以根本用不了 css
